tests/test_cli.py

```diff
@@ -353,7 +367,188 @@ def test_mapping_to_jinja():
 
     result = runner.invoke(
         mapping_to_jinja_cli,
-        ['-', '-', '--type', type_name],
+        ['-', '-', '--es_type', type_name],
+        input=json.dumps(mapping),
+    )
+
+    assert_no_exception(result)
+    # use the child template to modify the output template.
+    jinja_env = jinja2.Environment(
+        loader=jinja2.DictLoader({'generated': result.output}))
+    gen_output = json.loads(jinja_env.from_string(child_template).render())
+    assert gen_output == expected
+
+
+def test_multiple_mapping_to_jinja():
+    """Test mapping_to_jinja."""
```

Comment:
As said in the next comments, an explicit comment would be great. Maybe something like:
```
"""Test mapping_to_jinja with a valid mapping containing multiple types in a "mappings" dict."""
```